client/reader/stream/index.jsx

```diff
@@ -496,12 +496,23 @@ class ReaderStream extends React.Component {
 		return 'feed-post-' + ( postKey.feedId || postKey.blogId ) + '-' + postKey.postId;
 	}
 
-	handleConnectedCardClick = post => showSelectedPost( {
-		postKey: {
-			postId: post.feed_item_ID,
-			feedId: post.feed_ID,
+	handleConnectedCardClick = post => {
+		const postKey = {};
```

Comment:
i'm not a fan of how much postKey fidgeting we have.
Once the dust settles after CC maybe we could try to come up with a way where its less necessary